src/libsystemd-network/sd-dhcp6-lease.c

```diff
@@ -9,6 +9,7 @@
 #include "dhcp6-internal.h"
 #include "dhcp6-lease-internal.h"
 #include "network-common.h"
+#include "set.h"
```

Comment:
Please include `set.h` in dhcp6-lease-internal.h, rather than sd-dhcp6-lease.c, as the header requires it.

Reply:
Moved set.h to dhcp6-lease-internal.h